srcpkgs/gron/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'gron'
+pkgname=gron
+version=0.6.0
+revision=1
+build_style=go
+go_import_path=github.com/tomnomnom/gron
+hostmakedepends="git"
+short_desc="Tool to make JSON greppable"
+maintainer="Noel Cower <ncower@gmail.com>"
+license="MIT"
+homepage="https://${go_import_path}"
```

Comment:
Just write out the homepage, otherwise e.g. void-updates will literally `https://${go_import_path}` as homepage.